dbms/src/Flash/ResourceControl/TokenBucket.cpp

```diff
@@ -95,8 +98,8 @@ double TokenBucket::getDynamicTokens(const TokenBucket::TimePoint & timepoint) c
 {
     RUNTIME_CHECK(timepoint >= last_compact_timepoint);
     auto elspased = timepoint - last_compact_timepoint;
-    auto elapsed_second = std::chrono::duration_cast<std::chrono::seconds>(elspased).count();
-    return elapsed_second * fill_rate;
+    auto elapsed_ms = std::chrono::duration_cast<std::chrono::milliseconds>(elspased).count();
+    return static_cast<double>(elapsed_ms * fill_rate) / 1000;
```

Comment:
It looks like `/ 1000` doesn't need to be called multiple times.
`fill_rate_ms = fill_rate/1000`

Reply:
done